README.md

```diff
@@ -1,2 +1,12 @@
 # fcrepo-module-auth-webac
-WebAC Authorization Delegate Module for the Fedora 4 Repository
+WebAC Authorization Delegate Module for the Fedora 4 Repository. This is an implementation of the W3C's WebAccessControl draft specification at: [https://www.w3.org/wiki/WebAccessControl](https://www.w3.org/wiki/WebAccessControl).
+
+This module enables an ACL based access control to the fedora repository. Each protected resource should have a ACL file associated with it either directly or via an ancestor. The ACL file defines authorization based on three entities: 1) who has access, 2) what are the access permissions, and 3) to which resource. Please refer the above W3C specification for the ontology of the ACL RDF file.
+
+### Fedora 4 WebAC documentation
+[https://wiki.duraspace.org/display/FEDORA4x/WebAC+Authorization+Delegate](https://wiki.duraspace.org/display/FEDORA4x/WebAC+Authorization+Delegate)
+
+### Deployment and Configuration
+The [fcrepo-webapp-plus](https://github.com/fcrepo4-exts/fcrepo-webapp-plus) provides a convenient option to build fcrepo-webapp that is bundled with the WebAC module. Refer to the [fcrepo-webapp-plus README](https://github.com/fcrepo4-exts/fcrepo-webapp-plus/blob/master/README.md) for instructions. 
+
+The "Example Scenarios" section in the [WebAC Authorization Delegate wiki page](https://wiki.duraspace.org/display/FEDORA4x/WebAC+Authorization+Delegate) has configuration instructions for some of the common authorization scenarios.
```

Comment:
This is a super picky point, but you could link directly to this section with an anchor
```
https://wiki.duraspace.org/display/FEDORA4x/WebAC+Authorization+Delegate#WebACAuthorizationDelegate-ExampleScenarios
```
Otherwise it looks good. 
:+1: